<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton30">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>901</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбрать данные для электронного рабочего пространства:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton33">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>621</width>
     <height>16777215</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Данные</string>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton333">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>640</y>
     <width>431</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать электронное рабочее пространство</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
